npm/private/npm_import.bzl

```diff
@@ -673,20 +660,16 @@ def _npm_import_rule_impl(rctx):
                 bin_fields = "\n".join(bin_fields),
             ))
 
-            # Duplicate the bzl+BUILD into each linking package
-            for link_package in rctx.attr.link_packages.keys():
```

Comment:
Breaking change 2: each `npm__{pkg}__{version}` repo now has one `package_json.bzl` in the root. It is no longer duplicated into magic directories parallel to the pnpm workspace project directories.

If you want to import this package it is recommended to do `@npm//your/workspace/project:{pkg}/package_json.bzl` instead.